PULL_REQUEST_TEMPLATE

```diff
@@ -8,8 +8,6 @@ Closes # (issue number)
 
 ## Type of change
 
-Please delete options that are not relevant.
```

Comment:
I think we should keep this?

Reply:
yeah idk, I found it unnecessary seeing as how there's checkboxes and we can just check the option that's relevant. An alternative is to turn these (specifically the type of change) into list items instead of checkboxes and then just delete the item we want, but i think having it as checkboxes is nicer since you can just click to check them on github

Reply:
Here's an example of what I mean:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x]  New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This requires a run of yarn install

vs

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update
- This requires a run of yarn install

I just think having both options is a little silly